.NET/Microsoft.Recognizers.Text.DateTime/Utilities/StringUtil.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Recognizers.Text.DateTime.Utilities
+{
+    internal class StringUtil
+    {
+        public static string RemoveDiacritics(string str)
```

Comment:
In a next PR, can you move this to the base Text package. Such methods will be useful not only in DateTime.